tools/corto/src/cortotool_build.c

```diff
@@ -293,7 +357,9 @@ corto_int16 cortotool_coverage(int argc, char *argv[]) {
 
 corto_int16 cortotool_rebuild(int argc, char *argv[]) {
 
-    cortotool_generateRakefile();
+    if (cortotool_clean(argc, argv)) {
+        goto error;
+    }
 
```

Comment:
It shows the lines as added because of a change I did. That is what I mean by silent conflict. I'll fix it.